assignments/week05/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <link rel="stylesheet" href="css/style.css" />
+        <title>Assignment 05 - CSS Grid</title>
+    </head>
+    <body>
+<!-- Grid wrapper for all content page-->
+        <main class="main">
+<!-- Adding Header -->
+        <header class="header">
+            Header
+        </header>
+ <!-- Adding Nav bar -->
+        <nav class="nav">
+            <ul class="nav-ul">
+                <li>Nav Item</li>
+                <li>Nav Item</li>
+                <li>Nav Item</li>
+                <li>Nav Item</li>
+                <li>Nav Item</li>
+            </ul>
+        </nav>
+<!-- Adding Aside (Sidebar) -->
+        <aside class="aside">
+                <br/><br/><br/><br/><br/><br/>
```

Comment:
`<br>` are only meant for breaking lines in text, you can use flexbox here to achieve vertical alignment.